### ISSUE TITLE
Quick Notice: Language Selection Is not Accessible On Startup

### ISSUE THREAD

**Issue by lbk2907** [BearWare/TeamTalk5#1369]
### Description

Hey there. Just a quick notice. I’m current using TeamTalk 5.10.0.5086 – Unreleased. When it prompt me to choose the language, it was not read by default. I need to press NVDA+Tab or I need to press tab to go to the OK button and press Shift+Tab to back to the list each time I press an arrow key to choose the language. It doesn’t read anything when I use an arrow key to choose the language. This happened only on startup screen. Thanks.

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkClassic
- [ ] TeamTalkServer

### Platform

- [X] Windows
- [ ] macOS
- [ ] Android
- [ ] iOS
- [ ] Linux

### Actual behavior

Language Selection Was Not read Out By Screen Reader When Using Arrow Keys

### Steps to reproduce problem

Happend On StartUp Screen

**Comment by bear101** [comment]
@CoBC Is this an issue you can confirm?

**Comment by lbk2907** [comment]
I still facing this issue on version 5.10.0.5087

**Comment by CoBC** [comment]
I partially confirm this, here, when I start TT without config, when this dialog appears, NVDA says: "Choose language", but if I press arrow keys languages available are correctly read. The fact that screenreader doesn't always read focused item in a dialogbox like this has been fixed directly by Qt: https://bugreports.qt.io/browse/QTBUG-101585

**Comment by lbk2907** [comment]
@bearware101 @CoBC
In that dialog, I need to press NVDA+Tab to read the selected language.
For any tree view and list, it hard to get focus on it especially only have
1 item in that tree view or list.
Example, I can't focus on the file list if the channel only have 1 file
except for the first time I focus on the list when it available.
Another example is every time I open TeamTalk, log in to server, press Tab
once to the master volume, NVDA will correctly focus on it. I press
Shift+Tab to back to the channel tree view, NVDA will still correctly focus
on it. If I repeat the step above (press Tab and Shift+Tab) I need to press
up or down arrow once to make it move to another channel before it will
gain focus again.
That's why I said that it hard to gain focus on the tree view or list if
the tree view or list only have 1 item because when you press an arrow key,
it not have other item in the tree view or list that it can move to before
it can gain focus again.

**Comment by CoBC** [comment]
The issue with focus in trees has been fixed also by Qt, change will be 
available soon in TeamTalk.

**Comment by lbk2907** [comment]
@CoBC Thanks a lot. I'm waiting this for a long time.
